docs/cmd/tkn_pipeline_sign.md

```diff
@@ -36,6 +36,7 @@ or using kms
   -o, --output string                 Output format. One of: (json, yaml, name, go-template, go-template-file, template, templatefile, jsonpath, jsonpath-as-json, jsonpath-file).
       --show-managed-fields           If true, keep the managedFields when printing objects in JSON or YAML format.
       --template string               Template string or path to template file to use when -o=go-template, -o=go-template-file. The template format is golang templates [http://golang.org/pkg/text/template/#pkg-overview].
+  -v, --version string                apiVersion of the Pipeline to be signed (default "v1")
```

Comment:
usually on all clis, -v mean --verbose or to see the version of the tool, not to set version 

what about having --pipeline-version, this probably not going to be used often and at least until v1beta1 is completely deprecated./..

Reply:
Thanks that's a great suggestion! 😄